src/store/modules/searchclient.js

```diff
@@ -7,27 +7,27 @@ let sClient = {
     import.meta.env.VITE_ALGOLIA_API_READ_KEY,
     { _useRequestCache: true }
   ),
-  indexName: import.meta.env.VITE_ALGOLIA_INDEX_LAST_UPDATED_REPLICA,
+  indexName: import.meta.env.VITE_ALGOLIA_INDEX,
   availableIndexes: [
     {
-      value: import.meta.env.VITE_ALGOLIA_INDEX_LAST_UPDATED_REPLICA,
+      value: import.meta.env.VITE_ALGOLIA_INDEX,
       default: true,
-      orderedBy: 'updated',
-      isReplica: true,
-      label: 'Recently updated'
+      orderedBy: 'relevance',
+      isReplica: false,
+      label: 'Relevance'
```

Comment:
The new default index will be sorted by 'relevance' -- a combination of Algolia's default ranking and our own business logic (recommendations, most clicked books, presence of other desirable metadata attributes, etc.) as described in https://github.com/pressbooks/pressbooks-book-directory-fe/issues/319#issuecomment-866300771